src/pages/migrating/guide/design.mdx

```diff
@@ -155,55 +158,132 @@ the pixel/rem value that it is paired with.
 | Extra large (xl)         | 64 / 4            |
 | Double extra large (2xl) | 80 / 5            |
 
-### Color tokens
+## Type tokens <Tag type="cyan">Breaking</Tag>
 
-Existing color tokens have been renamed to better reflect their usage. In
-addition to renaming existing tokens, new tokens have been added to fill
-existing color token gaps in our system and to fix complex layering logic.
+The two v10 type sets—Productive and Expressive—have been blended together to
+work as a unified collection in v11. As a result of this convergence, type
+tokens name have been renamed to better define their relationship to one another
+and reflect its styling. For more information on the evolution of productive and
+express, see the [Typography page](/guidelines/typography/styling-strategies).
```

Comment:
"For more information on the evolution of productive and express,...". I think you cut off your sentence here.